dh/sidh/internal/p434/arith_amd64.s

```diff
@@ -22,52 +22,42 @@
 
 // Performs schoolbook multiplication of 128-bit with 256-bit
 // number. Uses MULX, ADOX, ADCX instruction.
-#define MULX128x256(IDX, M0, M1, T0, T1, T2, T3, T4, T5, T6)    \
-    XORQ    AX, AX              \
-    MOVQ    (IDX)(M0), DX       \
-    MULXQ   M1+ 0(SB), T0, T1   \ // T0 <- C0
+#define MULX128x256(I0, I1, M1, T0, T1, T2, T3, T4, T5)    \
```

Comment:
It would be helpful to indicate which of these parameters are inputs and outputs and what they mean.

Reply:
removed unused params